src/scss/constants.scss

```diff
@@ -66,6 +66,7 @@ $z-layers: (
   // Audience
   audience-emoji-reaction: 1000,
   audience-video: 2,
+  audience-participant-add-button: 1,
```

Comment:
[NB] I wonder if instead of setting this z-index, if we set `-1` on the overlay.. would that fix our problems here..?

Reply:
@0xdevalias I tried doing that. You are not able to click the buttons on the reaction bar, if you do that

Reply:
_nods_ fair.